index.html

```diff
@@ -22,5 +24,44 @@
       </div>
       <a href="">Download</a>
     </nav>
+    <section class="askedQuestion">
+      <div class="askedQuestion_left">
```

Comment:
and here then `asked-questions__left`  note double `__` before the element according to BEM we are following